accelerator/core/apis.c

```diff
@@ -672,3 +672,21 @@ status_t api_get_identity_info_by_id(const db_client_service_t* const db_service
   return ret;
 }
 #endif
+
+status_t api_check_iri_connection(const iota_client_service_t* const service, char** json_result) {
+  status_t ret = SC_OK;
+
+  ret = ta_check_iri_connection(service);
+  if (ret != SC_CCLIENT_FAILED_RESPONSE || ret != SC_CONF_IRI_BEHIND || ret != SC_OK) {
```

Comment:
This statement will always be true.

Reply:
Then `api_check_iri_connection` will always fail.

Reply:
just solved this